tests/test_client.py

```diff
@@ -2,10 +2,17 @@
 Test Client
 """
 from .hstestcase import HSTestCase
+from hubstorage import HubstorageClient
 from hubstorage.utils import millitime, apipoll
 
 class ClientTest(HSTestCase):
 
+    def test_custom_ua(self):
+        c = HubstorageClient(auth=cls.auth,
+                             endpoint=cls.endpoint,
+                             useragent='testUA')
+        self.assertEqual(c.useragent == 'testUA')
```

Comment:
Maybe another assertion to cover the case where the user doesn't pass in a custom value for `useragent`?